### PR TITLE
Make array type importing support multiple data

### PR DESCRIPTION
**Pull request by alexzhaobin**

### CODE REVIEW

File: docs/sdk/browser/android/ebates.md [hunk 1]
```diff
@@ -6,12 +6,13 @@ excerpt: Full instructions for integrating the Fillr Embedded library for Ebates
...
+First, integrate the Fillr SDK by following the instructions here [here](http://fillr.github.io/docs/sdk/browser/android/). You can skip section Fillr Toolbar Layout.
+
+Next, integrate the Fillr Embedded SDK by following the instructions here [here](http://fillr.github.io/docs/sdk/browser/android/embedded)
    
-### Create custom autofill button on your toolbar
+### Place custom autofill button on your toolbar
 
-If your app has an existing toolbar, you can also create custom autofill button and position it within your toolbar. This custom autofill button needs to be an instance of com.fillr.browsersdk.FillrAutofillButton class. Pass this autofill button to Fillr instance as follow, Fillr will control its visibility based on whether there is any fillable form.   
+If your app has an existing toolbar, you can create your custom autofill button and place it within your toolbar. This custom autofill button needs to be an instance of com.fillr.browsersdk.FillrAutofillButton class. Pass this autofill button to Fillr instance as follow, Fillr will take care of the rest and control its visibility based on whether there is any fillable form.
```
Comment (mhussain) [inline docs/sdk/browser/android/ebates.md]: `Pass this autofill button to Fillr instance as follow` -> Perhaps better would be `as below`

File: docs/sdk/browser/android/ebates.md [hunk 2]
```diff
@@ -34,7 +35,7 @@ if (autofillButton != null) {
 
 ### Importing user data from your app to Fillr
 
-To import user data, create an instance of com.fillr.userdataaccessor.UserDataAccessor class, passing data to it by calling importing methods. Everytime a import method is called, it will clean the related data in Fillr first, so the importing methods should only be called once. For cell phone, telephone, credit card, address, you need to create and pass in a corresponding Fillr user data model instance as follow:
+To import user data, create an instance of com.fillr.userdataaccessor.UserDataAccessor class, pass data to it by calling importing methods. Everytime an importing method is called, it will wipe the related data in Fillr first. For example, if you import email addresses, it will remove all existing emails in Fillr profile first. So the importing methods should only be called once when Fillr first initialized. For cell phone, telephone, credit card, address, you need to create and pass in a corresponding Fillr user data model instance as follow:
```
Comment (mhussain) [inline docs/sdk/browser/android/ebates.md]: Replace `,` with `and` at `create an instance of com.fillr.userdataaccessor.UserDataAccessor class,`. 

Possible `Everytime an importing method` should be `Every time an import method`

`only be called once when Fillr first initialized` -> `only be called once, when Fillr is first initialized`

`For cell phone, telephone, credit card, address,` -> `For cell phone, telephone, credit card and address fields`

File: docs/sdk/browser/android/ebates.md (hunk 1 above)
Comment (alexzhaobin) [inline docs/sdk/browser/android/ebates.md]: Updated

File: docs/sdk/browser/android/ebates.md (hunk 2 above)
Comment (alexzhaobin) [inline docs/sdk/browser/android/ebates.md]: Updated

**Pull request merged**